cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_on_termination_script.sh.erb

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+DEFAULT_USER=''
+OS=$(cat /etc/os-release | grep '^ID=' | cut -f2 -d'"')
+TERMINATION_MESSAGE=$1
+GRACETIME_PERIOD=$2
+
+if [[ $OS == "amzn" ]]; then
+    DEFAULT_USER='ec2-user'
+elif [[ $OS == "centos" ]]; then
+    DEFAULT_USER='centos'
+elif [[ $OS == "ubuntu" ]]; then
+    DEFAULT_USER='ubuntu'
+elif [[ $OS == "rhel" ]]; then
+    DEFAULT_USER='ec2-user'
+fi
```

Comment:
You can omit this logic and inject the cluster user name from the node attribute `node['cluster']['cluster_user']`

Reply:
Thank you for your reminding. Done!